app/mailers/proposals_answers_mailer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ProposalsAnswersMailer < ApplicationMailer
+  include Decidim::SanitizeHelper
+  include Decidim::ComponentPathHelper
+
+  layout "decidim/mailer"
+
+  def notify_massive_import(answers_collection, user)
+    @organization = user.organization
+    @participatory_space_title = decidim_sanitize_translated(answers_collection.first.participatory_space.title) if answers_collection.first.present?
```

Comment:
extract `answers_collection.first` to a variable and avoid doing multiple times.

Reply:
Yes, you're right. Thanks!